cognite/replicator/replication.py

```diff
@@ -194,6 +201,9 @@ def make_objects_batch(
                 update_objects.append(dst_obj)
             else:
                 unchanged_objects.append(dst_obj)
+        elif dst_ts:
+            if any(x for x in dst_ts_ext_id_set if x == src_obj.external_id):
```

Comment:
I think this line can now be replaced with
```python
if src_obj.external_id in dst_ts_ext_id_set:
```

Reply:
It is a bit late so hard to focus on the logic right now, but what if dst_ts is given, but the if is not triggered, shouldn't we create a new object then?

Reply:
i think we dont need to
Basically we need dst_ts only for timeseries that are present in the destination tenant but have not been copied by the replicator (they will not have the fields _replicatedxxxx, so will not be present in the map src_id_dst_map)
If not found in the map, the replicator would try to create a timeseries, but it would fail
If the timeseries series is not found in the map, before creating it, we check if it does not exist already in the destination and if it exists we do not touch it (add to unchanged_objects)
Hope that makes sense

Reply:
Makes sense!